roles/kubernetes-apps/csi_driver/vsphere/templates/vsphere-csi-controller-deployment.yml.j2

```diff
@@ -99,7 +99,7 @@ spec:
               name: socket-dir
 {% endif %}
         - name: vsphere-csi-controller
-          image: {{ gcr_image_repo }}/cloud-provider-vsphere/csi/release/driver:{{ vsphere_csi_controller }}
+          image: registry.k8s.io/csi-vsphere/driver:{{ vsphere_csi_controller }}
```

Comment:
HI @luringens 

We can use `{{ kube_image_repo }}` instead of `registry.k8s.io`